README.md

```diff
@@ -489,8 +489,8 @@ Troubleshooting
 ---------------
 
 If you have problems building Sage, check the Sage Installation Guide,
-as well as the version-specific Sage Installation FAQ in the [Sage Release
-Tour](https://wiki.sagemath.org/ReleaseTours) corresponding to the
+as well as the version-specific Sage Installation FAQ in the [release
+tour](https://github.com/sagemath/sage/releases) corresponding to the
 version that you are installing.
```

Comment:
There is no "Installation FAQ". How about "the version-specific installation help in the ..."?